content/tutorials/dev-guide.md

```diff
@@ -72,19 +70,19 @@ Controls:
 
 ## Object callbacks
 
-One way to reduce the flood of messages for your Python program is to define a callback specifically for one object, our cube for example. Update your program to comment out all messages subscribed in the scene, add a callback just for your cube object, and update the creation of your cube object with a click-listener (`clickable=True`) and the new `cube_callback`.
+One way to reduce the flood of messages for your Python program is to define a callback specifically for one object, our box for example. Update your program to comment out all messages subscribed in the scene, add a callback just for your box object, and update the creation of your box object with a click-listener and the new `box_callback`.
 
 ```python
-def scene_callback(msg):
-#    print("scene_callback: ", msg)
+def on_msg_callback(obj):
+#    print("scene_callback: ", obj)
 
-def cube_callback(msg):
-    print("cube_callback: ", msg)
+def box_callback(evt):
+    print("box_callback: ", evt)
 
-arena.Object(objType=arena.Shape.cube, clickable=True, callback=cube_callback)
+box = Box(evt_handler=box_callback)
```

Comment:
You still need `clickable=True` for evt_handlers.
Should be
`box = Box(clickable=True, evt_handler=box_callback)`